Frontend-v1-Original/components/options/stake.tsx

```diff
@@ -227,35 +224,40 @@ export function Stake() {
           <div>Total staked</div>
           <div>${formatCurrency(totalStakedValue)}</div>
         </div>
-        <div className="flex items-center justify-between">
-          <div className="flex items-center justify-start gap-2">
-            <div>APR</div>
-            {displayedRewardTokens && displayedRewardTokens.length > 0 && (
-              <div className="flex items-center justify-center gap-1">
-                {displayedRewardTokens.map((token) => (
+        <div className="my-1 flex items-center justify-between">
```

Comment:
```suggestion
        <div className="flex items-start justify-between">
```
I would remove `my-1` here to keep consistency with others.

As for horizontal alignment, I would prefer `items-start` in this case. But it's up to you.

<img width="525" alt="image" src="https://github.com/Velocimeter/frontend/assets/126733611/d10a0ea3-5e69-40be-83ff-212bcaa824d3">